src/libcore/num/flt2dec/mod.rs

```diff
@@ -410,8 +410,8 @@ fn determine_sign(sign: Sign, decoded: &FullDecoded, negative: bool) -> &'static
 /// it will only print given digits and nothing else.
 ///
 /// The byte buffer should be at least `MAX_SIG_DIGITS` bytes long.
-/// There should be at least 5 parts available, due to the worst case like
-/// `[+][0.][0000][45][0000]` with `frac_digits = 10`.
+/// There should be at least 4 parts available, due to the worst case like
+/// `[+][0.][0000][2][0000]` with `frac_digits = 10`.
```

Comment:
… I think you are overthinking about this 😄 [When the documentation was originally written](https://github.com/lifthrasiir/rust-strconv/blob/0c5df0ea184482c40d37e8553192c563570db211/src/flt2dec/mod.rs#L478-L490), the `+` sign is indeed a part, so `[+][1][.][2345][e][-][67]` had 7 parts because there are 7 `[]`s, not due to 1234567. Later [the `+` sign is no longer a part](https://github.com/lifthrasiir/rust-strconv/commit/f65f6445b389f33e3a455b81ed918eea585e5f9d) but the documentation failed to catch up until this PR.